Makefile

```diff
@@ -33,6 +33,7 @@ vet:
 
 build:
 	@echo ">> building binaries"
+	@./build/assets.sh
```

Comment:
Alternatively, you could update the script to only run if the modified time on source files is more recent than the output files.